atd-vzd/views/view_fatalities.sql

```diff
@@ -0,0 +1,45 @@
+-- view of fatalities in jursidiction ID with crash attributes joined
+CREATE OR REPLACE VIEW view_fatalities AS (
+    WITH fatalities_plus_crash AS (
+        SELECT
+            f.id,
+            f.crash_id,
+            f.victim_name,
+            f.law_enforcement_num,
+            f.ytd_fatal_crash,
+            f.ytd_fatality,
+            f.person_id,
+            f.primaryperson_id,
+            TO_CHAR(crashes.crash_date,
+                'yyyy') AS year,
+            CONCAT_WS(' ',
+                crashes.rpt_block_num,
+                crashes.rpt_street_pfx,
+                crashes.rpt_street_name,
+                '(',
+                crashes.rpt_sec_block_num,
+                crashes.rpt_sec_street_pfx,
+                crashes.rpt_sec_street_name,
+                ')') AS location,
+            crashes.position,
+            crashes.crash_date,
+            crashes.case_id,
+            f.is_deleted
+        FROM
+            fatalities f
+        LEFT JOIN atd_txdot_crashes crashes ON f.crash_id = crashes.crash_id
+)
+SELECT
+    *
+FROM
+    fatalities_plus_crash
+WHERE
+    ST_CONTAINS((
+        SELECT
+            geometry FROM atd_jurisdictions
+        WHERE
+            atd_jurisdictions.id = 5),
+        fatalities_plus_crash.position)
+AND 
+    fatalities_plus_crash.is_deleted = false
+);
```

Comment:
to recap recent conversation—if we were confident in the `atd_txdot_crashes.austin_full_purpose` column, we could do away with this spatial filter. also be advised that this filter uses `atd_jurisdictions.id = 5` while the `austin_full_purpose` column includes [a few additional jurisdictions](https://github.com/cityofaustin/atd-vz-data/blob/53ddfccd3c24b81a9a35ed4af766d68ca02cd971/atd-events/crash_update_jurisdiction/app.py#L158).

Reply:
For posterity—my previous comment is incorrect. Refer to [this thread](https://austininnovation.slack.com/archives/CM9MK950S/p1678398208097029) for more background on the austin_full_purpose field.